### DISCUSSION TITLE
Submit error; Err: '[161280] not in index'! ; team name: tteam

### DISCUSSION THREAD

**Discussion by h3z** [PaddlePaddle/PaddleSpatial#223] [BaiduKDDCup2022] [answered]
<img width="1093" alt="image" src="https://user-images.githubusercontent.com/28476985/175462492-308c586c-bf47-4ce9-92d5-4075e923889f.png">

**Comment by ethanyxx** [comment, marked as answer]
Hi,
Below is the log info related to your submission error. 
<img width="956" alt="截屏2022-06-24 17 27 57" src="https://user-images.githubusercontent.com/62667125/175507061-c7bfaee2-29e6-44b3-a0a9-d4ef1e683ad7.png">

**Reply by h3z** [reply]
thanks
